### DISCUSSION TITLE
Why is it not converged at low temperatures in SCPH?

### DISCUSSION THREAD

**Discussion by gangliu-github** [ttadano/alamode#201] [Q&A] [answered]
I performed the SCPH calculation. It is not converged at low temperatures (0K, 100K, 200K), while it achieved convergence at high temperatures (300~1200K). 
Why? How can I improve the convergence of SCPH?

**Comment by ttadano** [comment, marked as answer]
A potential solution is to use a smaller `MIXALPHA` and a smaller `DT`.

**Reply by gangliu-github** [reply]
Thanks! I will try.

**Reply by gangliu-github** [reply]
I have tried the solution, and it improved the convergence really! Thank you!
